backend/src/main/game/cards/playable_card_logic.py

```diff
@@ -0,0 +1,50 @@
+
+class PlayableCardLogic:
+    def __init__(self, hand):
+        self.hand = hand
+        self.discard = []
+        self.lost = []
+        if not self.hand:
+            raise ValueError('wrong input value')
```

Comment:
Use a more meaningful message. e.g. `Card Logic requires a non-empty hand to be passed in`